docs/devices/pressure/EPC.md

```diff
@@ -0,0 +1,28 @@
+# Bronkhorst El-Pressure controller
+
+## Introduction
+
+Bronkhorst El-Pressure controller controller (EPC) device driver.
+
+More information about the device can be found in the supplier platform [Bronkhorst](https://www.bronkhorst.com/en-gb/).
+
+## mass flow controller
+
+
+
+## Configuration
+Configuration sample showing all possible parameters:
+
+```toml
+[device.my-EPC]  # This is the MFC identifier
+port = "COM4"    # Access port (serial)
+channel = 1      
+address = 0x80
+max_pressure = 9
+```
```

Comment:
This part needs to be improved. Explain what is necessary and what is not. (please research what is needed for all devices we have).